libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/FeaturesUtils.ts

```diff
@@ -0,0 +1,13 @@
+import { localization } from "@responsible-ai/localization";

```

Comment:
looks like there are some linting errors, you can run yarn lintfix to autofix them or see those which can't be automatically fixed